browser/app/js/objects/ObjectsBulkActions.js

```diff
@@ -47,69 +64,136 @@ export class ObjectsBulkActions extends React.Component {
+        </svg>
+      ),
+      download: (
+        <svg width="12" height="14">
+          <path d="M11.338 5.088H8.195V.5H3.48v4.588H.338l5.5 5.353 5.5-5.353zm-11 6.883v1.53h11v-1.53h-11z" />
+        </svg>
+      ),
+      trash: (
+        <svg width="11" height="14">
+          <path d="M1.214 12.022c0 .813.643 1.478 1.43 1.478h5.714c.786 0 1.43-.665 1.43-1.478v-8.87H1.214v8.87zM10.5 1.163H8L7.286.5H3.714L3 1.163H.5V2.5h10V1.163z" />
+        </svg>
+      ),
+      close: (
+        <svg width="12" height="12">
+          <path d="M11.316 1.948L10.318.95l-3.96 3.96L2.398.95l-.998.998 3.96 3.96-3.96 3.96 1 1 3.96-3.96 3.96 3.96 1-1-3.96-3.96z" />
+        </svg>
+      )
+    }
+
     return (
-      <div
-        className={
-          "list-actions" +
-          classNames({
-            " list-actions-toggled": checkedObjects.length > 0
-          })
-        }
-      >
-        <span className="la-label">
-          <i className="fa fa-check-circle" /> {checkedObjects.length}
-          {checkedObjects.length === 1 ? " Object " : " Objects "}
-          selected
-        </span>
-        <span className="la-actions pull-right">
-          <button id="download-checked" onClick={this.handleDownload.bind(this)}>
-            {" "}
-            Download
-            {(checkedObjects.length === 1 && !checkedObjects[0].endsWith('/')) ? 
-            " object" : " all as zip" }{" "}
-          </button>
-        </span>
-        <span className="la-actions pull-right">
+      <React.Fragment>
+        <div
+          className={classNames({
+            "object-actions": true,
+            "object-actions--active": checkedObjectsCount
+          })}
+        >
           <button
             id="delete-checked"
-            onClick={() => this.setState({ showDeleteConfirmation: true })}
+            className="object-actions__item"
+            onClick={() =>
+              this.setState({
+                showDeleteConfirmation: true
+              })
+            }
+            disabled={!checkedObjectsCount}
           >
-            {" "}
-            Delete selected{" "}
+            {icons.trash}
+            <span>Delete selected</span>
           </button>
-        </span>
-        <i
-          className="la-close fa fa-times"
-          id="close-bulk-actions"
-          onClick={clearChecked}
-        />
+          <button
+            id="download-checked"
+            className="object-actions__item"
+            onClick={this.handleDownload.bind(this)}
+            disabled={!checkedObjectsCount}
+          >
+            {icons.download}
+            <span>
+              Download {checkedObjects.length > 1 ? " all as zip" : " object"}
```

Comment:
We would also need to check if the checkedObject is a directory or not as when I select a directory, the download message is as follows:

![screen shot 2018-09-27 at 11 36 56 am](https://user-images.githubusercontent.com/27314853/46167072-ad423a80-c249-11e8-95db-70d1b2d7ba6d.png)